src/main/java/graphql/schema/GraphQLFieldDefinition.java

```diff
@@ -185,11 +163,7 @@ public Builder deprecate(String deprecationReason) {
 
         public GraphQLFieldDefinition build() {
             if (dataFetcher == null) {
-                if (isField) {
-                    dataFetcher = new FieldDataFetcher(name);
-                } else {
-                    dataFetcher = new PropertyDataFetcher(name);
-                }
+                dataFetcher = new PropertyDataFetcher(name);
```

Comment:
Yes. +1 to no default. Never liked em hidden defaults.

My question however was more from the code change perspective. If isField then instantiate a FieldDataFetcher. Else a PropertyDataFetcher. Now you've removed the isField flag which tells me that this is always going to be a field. Right? And by that logic it should always _sigh_ default to FieldDataFetcher. Did I miss something?

Still maintain that it shouldn't default but I wonder what is the impact for those already in production with this.